src/ssh-util/src/tunnel_manager.rs

```diff
@@ -52,15 +51,18 @@ impl SshTunnelManager {
     pub async fn connect(
         &self,
         secrets_reader: &dyn SecretsReader,
-        tunnel: &SshTunnel,
+        connection_id: GlobalId,
+        host: &str,
+        port: u16,
+        user: &str,
```

Comment:
This hunk gets immediately refactored in the next commit. I think you should either rework how the commits are split up, or merge the two commits together.